source/framework/vx_reference.c

```diff
@@ -2313,6 +2313,7 @@ VX_API_ENTRY vx_status vxExtendSupplementaryUserDataObject(vx_reference destinat
         /* Destination cannot be supplementary data of another object */
         status = (vx_status)VX_ERROR_NOT_SUPPORTED;
     }
+    /* the combination "is_accessible" and "is_virtual" beeing different is untestable */
```

Comment:
Minor, but can we correct the typo here ("beeing" should be "being")

Reply:
fixed